### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by isamu**
Add descriptions of  functions:secrets:* of Cloud Functions Commands

### Description

I just added descriptions of functions:secrets:* of Cloud Functions Commands in README.md.
descriptions are from src/commands/functions-secrets-*.ts . Thanks.

### CODE REVIEW

**Comment by bkendall** [comment]
This seems pretty reasonable 😄  I'll let someone on the functions side of things just LGTM as well...

File: README.md [hunk 1]
```diff
@@ -136,15 +136,20 @@ Detailed doc is [here](https://firebase.google.com/docs/cli/auth).
...
-| **functions:config:get**   | Retrieve existing configuration values for the current project's Cloud Functions.                            |
-| **functions:config:unset** | Remove values from the current project's runtime configuration.                                              |
-| **functions:config:clone** | Copy runtime configuration from one project environment to another.                                          |
-| **functions:delete**       | Delete one or more Cloud Functions by name or group name.                                                    |
-| **functions:shell**        | Locally emulate functions and start Node.js shell where these local functions can be invoked with test data. |
+| Command                       | Description                                                                                                  |
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------ |
+| **functions:log**             | Read logs from deployed Cloud Functions.                                                                     |
```
Comment (colerogers) [inline README.md:141]: If you wanted to be exhaustive, we also have this command you could add here:
```
| **functions:list** | List all deployed functions in your Firebase project. |
```

**Review by colerogers — APPROVED** [review]
Looks good, thanks for your contribution!

File: README.md (hunk 1 above)
Comment (isamu) [reply README.md:141]: Thank you for your feedback.
I add functions:list in list.

**Review by bkendall — APPROVED** [review]

**Pull request merged**